queue_services/entity-bn/tests/unit/bn_processors/test_change_of_registration.py

```diff
@@ -12,42 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """The Test Suites to ensure that the change of registration is operating correctly."""
+from http import HTTPStatus
+import uuid
 import xml.etree.ElementTree as Et
 
 import pytest
 from legal_api.models import RequestTracker
 
 from entity_bn.bn_processors import bn_note
-from entity_bn.exceptions import BNException, BNRetryExceededException
-from entity_bn.worker import process_event
-from tests.unit import create_filing, create_registration_data
+from tests.unit import create_filing, create_registration_data, get_json_message
 
 
-@pytest.mark.parametrize('legal_type', [
-    ('SP'),
-    ('GP'),
-])
-async def test_change_of_registration(app, session, mocker, legal_type):
+message_type = "bc.registry.business.changeOfRegistration"
+
+
+@pytest.mark.parametrize(
+    "legal_type",
+    [
+        ("SP"),
+        ("GP"),
+    ],
+)
+def test_change_of_registration(app, session, client, mocker, legal_type):
     """Test inform cra about change of SP/GP registration."""
-    filing_id, business_id = create_registration_data(legal_type, tax_id='993775204BC0001')
+    identifier = "FM1234567"
+    filing_id, legal_entity_id = create_registration_data(
+        legal_type, identifier=identifier, tax_id="993775204BC0001"
+    )
     json_filing = {
-        'filing': {
-            'header': {
-                'name': 'changeOfRegistration'
-            },
-            'changeOfRegistration': {
-                'offices': {
-                    'businessOffice': {
-                        'mailingAddress': {},
-                        'deliveryAddress': {}
-                    }
+        "filing": {
+            "header": {"name": "changeOfRegistration"},
+            "changeOfRegistration": {
+                "offices": {
+                    "businessOffice": {"mailingAddress": {}, "deliveryAddress": {}}
                 },
-                'parties': [{}]
-            }
+                "parties": [{}],
+            },
         }
     }
-    filing = create_filing(json_filing=json_filing, business_id=business_id)
-    filing._meta_data = {'changeOfRegistration': {'toLegalName': 'new name'}}
+    filing = create_filing(json_filing=json_filing, legal_entity_id=legal_entity_id)
+    filing._meta_data = {"changeOfRegistration": {"toLegalName": "new name"}}
```

Comment:
`toLegalName` won't be used in the filing.meta_data anymore.  

The meta data will look like this:

```
{
  "legalFilings": [
    "changeOfRegistration"
  ],
  "applicationDate": "2023-04-26T19:01:12.794272+00:00",
  "changeOfRegistration": {
    "toBusinessName": "AC SP CON 2023.APR.26 18.46 TEST",
    "fromBusinessName": "AC SP 2023.APR.26 18.30 TEST"
  }
}
```